packages/core/src/Stores/notification-store.js

```diff
@@ -130,6 +137,17 @@ export default class NotificationStore extends BaseStore {
         );
     }
 
+    get is_notifications_loading_completed() {
+        const { is_switching, is_logging_in, is_landing_company_loaded, is_account_setting_loaded } =
+            this.root_store.client;
+
+        // check switching, logging, landing_company, and account_setting
+        if (is_switching || is_logging_in || !is_landing_company_loaded || !is_account_setting_loaded) return false;
+
+        // return false if at least one property is false
+        return !Object.values(this.notifications_updated).some(e => !e);
```

Comment:
@sergei-deriv let's try to make is_notifications_loading_completed an observable instead of computed and add a condition to the reaction so that we set is_notifications_loading_completed to true when it's complete inside a reaction in client-store.
I also suggest that we add self-documenting code and not one-letter variables like `e` here that will be readable without comments. If we need comments for data manipulations, we have to simplify them, rename variables and use positive conditions for readability👍🏻
 E.g. instead of this:
 // return false if at least one property is false
` !Object.values(this.notifications_updated).some(e => !e);`
 it's better to turn it to positive condition:
 // return true if all properties are true
` Object.values(this.notifications_updated).every(Boolean);`
But i still believe that we don't need to use the notifications_updated object at all and is_notifications_loading_completed can be a `boolean` observable set using an `action` that we can add here and call it from a reaction in client-store.

Reply:
@maryia-deriv Ok, I'll change it